cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -178,8 +178,14 @@ func generateMountsModeRecover(mst *initramfsMountsState, recoverySystem string)
 		return err
 	}
 	if !isRecoverDataMounted {
-		// TODO:UC20: data may need to be unlocked
-		fmt.Fprintf(stdout, "/dev/disk/by-label/ubuntu-data %s\n", boot.InitramfsHostUbuntuDataDir)
+		// TODO: do we do this in recover mode too ?
```

Comment:
I'm confused, the todo says "do we do this in recover mode too" but this is in generateMountsModeRecover - should the question be about "install" mode?

Reply:
sorry let me delete that TODO it is wrong